apps/src/templates/studioHomepages/ResourceCard.jsx

```diff
@@ -26,17 +26,22 @@ const styles = {
     paddingRight: 20,
     color: color.white
   },
+  titleContainer: {
+    width: '100%',
+    display: 'flex',
+    alignItems: 'baseline',
+    padding: 20,
+    paddingBottom: 15
+  },
   title: {
+    color: color.white,
     fontFamily: '"Gotham 7r", sans-serif',
-    paddingTop: 20,
-    paddingBottom: 15,
     fontSize: 27,
-    width: '100%',
-    display: 'inline',
+    lineHeight: '29px',
     boxSizing: 'border-box'
   },
   titleSmall: {
-    width: '100%',
+    // width: '100%',
```

Comment:
just noticed this. i'm going to wait for the build to finish before pushing the fix.